gatsby-config.js

```diff
@@ -54,9 +54,10 @@ module.exports = {
             },
         },
         {
-            resolve: `gatsby-transformer-remark`,
+            resolve: `gatsby-plugin-mdx`,
             options: {
-                plugins: [
+                extensions: [`.mdx`, `.md`],
```

Comment:
RE comment above

Reply:
yep sorry I wasn't clear enough before: with the screenshot I had meant to indicate that I'd included the file extension array in gatsby-config.js 👍🏻